baseframe/forms/fields.py

```diff
@@ -203,7 +203,9 @@ def __init__(self,
         tinymce_options.setdefault('plugins', "autolink autoresize link lists paste searchreplace")
         tinymce_options.setdefault('toolbar', "bold italic | bullist numlist | link unlink | searchreplace undo redo")
         tinymce_options.setdefault('width', "100%")
-        tinymce_options.setdefault('height', "400")
+        tinymce_options.setdefault('height', "200")
+        tinymce_options.setdefault('autoresize_min_height', "159")
+        tinymce_options.setdefault('autoresize_max_height', "200")
```

Comment:
All of the right-hand-side values should be in single quotes. This is pre-existing code, but might as well fix it now that this file is being modified.

Reply:
Fixed